### PR TITLE
graphs/gen: note the graph models for Gnm and Gnp

### LINKED ISSUE #191: graphs/gen/batagelj_brandes.go should be renamed to erdos_renyi
I think it would be more recognizable. This looks like the Erdos-Renyi model. Thoughts?

### PR DESCRIPTION
**Pull request by kortschak**
Fixes #191.

@jcao219 @vladimir-ch Please take a look.

### CODE REVIEW

**Comment by kortschak** [comment]
@jcao219 if that is an approval, would you please either comment "LGTM" or make an approval.

**Comment by jcao219** [comment]
LGTM

File: graphs/gen/batagelj_brandes.go [hunk 1]
```diff
@@ -16,10 +16,10 @@ import (
 	"github.com/gonum/graph/simple"
 )
 
-// Gnp constructs a graph in the destination, dst, of order n. Edges between nodes
-// are formed with the probability, p. If src is not nil it is used as the random
-// source, otherwise rand.Float64 is used. The graph is constructed in O(n+m) time
-// where m is the number of edges added.
+// Gnp constructs a Gilbert’s mode graph in the destination, dst, of order n. Edges
```
Comment (vladimir-ch) [inline graphs/gen/batagelj_brandes.go]: model graph?

Comment (kortschak) [inline graphs/gen/batagelj_brandes.go]: Thank you.

**Pull request merged**